examples/run_over_exo.sh

```
@@ -0,0 +1,68 @@
+    source "$HOME/.exoenv/bin/activate"
+    EXO_MODELS_READ_ONLY_DIRS="$HOME/.exo/lmstudio-links" \
+    command exo &
+    local exo_pid=$!
```

Shellcheck (0.11.0)

[error] 'local' is only valid in functions.

(SC2168)